### DISCUSSION TITLE
GitHub Releases and F-Droid presence

### DISCUSSION THREAD

**Discussion by I-I-IT** [a-ghorbani/pocketpal-ai#114] [Ideas]
Your tool is really amazing, plus it's open-source which is really nice. Could you publish the apk files on GitHub (Releases) and also on F-Droid? That would be nice to avoid a dependency on Big Tech platforms.

**Comment by a-ghorbani** [comment]
Regarding apk: I just finished setting up the CI today: [Release v1.5.27](https://github.com/a-ghorbani/pocketpal-ai/releases/tag/v1.5.27). From now on, every time we release a new version, the CI will publish the apk here too.

Regarding F-Droid: I don't have any experience with it. will need to look into it. But, if anyone is interested in taking this on, it would be appreciated!

**Reply by a-ghorbani** [reply]
related issue: #55
